network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -485,6 +544,14 @@ class Homepage(FoundationMetadataPageMixin, Page):
         ),
         InlinePanel('featured_blogs', label='Blogs', max_num=4),
         InlinePanel('featured_highlights', label='Highlights', max_num=5),
+        MultiFieldPanel(
+            [
+                FieldPanel('take_action_title'),
+                InlinePanel('take_action_cards', label='Take Action Cards', max_num=4),
+            ],
+            heading='Take Action',
+            classname='collapsible',
+        ),
```

Comment:
let's add the "collapsed by default" here, since this one takes up quite a lot of admin view real estate.